include/ck_tile/ops/gemm/warp/warp_gemm_attribute_smfmac.hpp

```diff
@@ -0,0 +1,66 @@
+#include "ck_tile/core.hpp"
+#include "ck_tile/ops/gemm/warp/warp_gemm_attribute_smfmac_impl.hpp"
+
+namespace ck_tile {
+
+template <typename WarpGemmAttributeSmfmacImpl_>
+struct WarpGemmAttributeSmfmac
```

Comment:
Please add some doc to this class. Describe all (if any) instruction assumptions.